src/app/jobseekers-login/jobseekers-login.component.html

```diff
@@ -0,0 +1,89 @@
+<div class="jobseeker-container mt-2">
+  <div class="col-12-xs col-7-md col-7-xl col-7-xxl">
+    <div class="card bg-white p-2">
+      <div class="card-content">
+        <div class="">
+          <h1 class="font-xxl font-semibold text-light-blue mb-1">
+            Jobseekers
+          </h1>
+          <p class="mb-1">Log in to your Jobseeker Account</p>
+          <div class="form-container">
+            <form [formGroup]="loginForm" (ngSubmit)="loginJobseeker()">
+              <div class="input-group mt-2">
+                <div class="col-12-xs col-12-md col-12-xl">
+                  <label for="email" class="font-md">Email</label>
+                  <input
+                    id="email"
+                    type="email"
+                    class="input"
+                    formControlName="email"
+                    [ngClass]="{
+                              'border-red':
+                                loginForm.controls.email.invalid &&
```

Comment:
Kindly use a getter to access the forms to make it much more readable. 
eg.  get userEmail(){
return this.loginForm.get('email)
}